modules/distr-tracing-tempo-deploy-streaming.adoc

```diff
@@ -0,0 +1,155 @@
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-deploy-streaming_{context}"]
+= Deploying the {DTShortName} streaming strategy from the web console
+
+The `streaming` deployment strategy is intended for production environments that require a more scalable and highly available architecture, and where long-term storage of trace data is important.
+
+The `streaming` strategy provides a streaming capability that sits between the Collector and the Elasticsearch storage. This reduces the pressure on the storage under high load situations, and enables other trace post-processing capabilities to tap into the real-time span data directly from the Kafka streaming platform.
+
+[NOTE]
+====
+The streaming strategy requires an additional Red Hat subscription for AMQ Streams. If you do not have an AMQ Streams subscription, contact your sales representative for more information.
+====
+
+[NOTE]
+====
+The streaming deployment strategy is currently unsupported on {ibmzProductName}.
+====
+
+.Prerequisites
+
+* The AMQ Streams Operator has been installed. If using version 1.4.0 or higher you can use self-provisioning. Otherwise you must create the Kafka instance.
+* The {TempoName} Operator has been installed.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `tracing-system`.
+
++
+[NOTE]
+====
+If you are installing as part of Service Mesh, the {DTShortName} resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource, for example `istio-system`.
+====
++
+
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `tracing-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `tracing-system` from the *Project* menu. You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the {TempoName} Operator. On the *Overview* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Tempo*, click *Create Instance*.
+
+. On the *Create Tempo* page, replace the default `all-in-one` YAML text with your streaming YAML configuration, for example:
+
+.Example tempo-streaming.yaml file
+[source,yaml]
+----
+apiVersion: tempotracing.io/v1
+kind: Tempo
+metadata:
+  name: tempo-streaming
+spec:
+  strategy: streaming
+  collector:
+    options:
+      kafka:
+        producer:
+          topic: tempo-spans
+          #Note: If brokers are not defined,AMQStreams 1.4.0+ will self-provision Kafka.
+          brokers: my-cluster-kafka-brokers.kafka:9092
+  storage:
+    type: elasticsearch
+  ingester:
+    options:
+      kafka:
+        consumer:
+          topic: tempo-spans
+          brokers: my-cluster-kafka-brokers.kafka:9092
+
+----
```

Comment:
This CR is for Jaeger. We need to review/adapt to Tempo

Reply:
Outdated. Resolving this thread.